client/src/content/home-en.js

```diff
@@ -43,6 +49,7 @@ The top 32 teams after qualifiers will then face off in a Group Stage, where 16
       title: "COEGTS 2023",
       fullTitle: "CAVOE'S OSU! EVENT'S GLOBAL TAIKO SHOWDOWN 2023",
       code: "coegts",
+      fullCode: "coegts_2023",
```

Comment:
this one should be the `ongoing` tourney rather than egts right?

Reply:
ya do pls change it to COEGTS being the main one

Reply:
Applied in 400d747fcda341fb788f49c155bcf464530e1af2